en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,122 @@
+# Traxxas Stampede VXL
+
+{% youtube %}https://youtu.be/N3HvSKS3nCw{% endyoutube %}
+
+![Traxxas Stampede VXL](../../assets/airframes/experimental/stampede/stampede.jpg)
+
+## Parts List
+
+  * [Traxxas Stampede](https://traxxas.com/products/models/electric/stampede-vxl-tsm) All of this is used except for the top plastic cover.
+  * [Pixhawk Mini](https://store.3dr.com/products/3dr-pixhawk)
+  * [Power Module](https://store.3dr.com/products/10s-power-module)
+  * [Telemetry Module (EU)](https://store.3dr.com/products/433-mhz-telemetry-radio) US version available in the website
+  * [Spektrum Dxe Controller](http://www.spektrumrc.com/Products/Default.aspx?ProdId=SPM1000) or other PX4-compatible remotes
+  * [Spektrum Quad Race Serial Receiver w/Diversity](http://www.spektrumrc.com/Products/Default.aspx?ProdID=SPM4648)
+  * [PX4Flow](https://pixhawk.org/modules/px4flow)
+
+
+## Assembly
+
+The assembly consists of a wooden frame on which all the parts were attached. Tests showed that a better vibration insulation should be used, especially for the Pixhawk and the Flow module.
+
+![Assembly0](../../assets/airframes/experimental/stampede/stampede_chassis.jpg)
+
+![Assembly1](../../assets/airframes/experimental/stampede/panel_top.png)
+
+![Assembly2](../../assets/airframes/experimental/stampede/panel_bottom.png)
+
+![Assembly3](../../assets/airframes/experimental/stampede/final_assembly.jpg)
+
+![Assembly4](../../assets/airframes/experimental/stampede/final_side.jpg)
+
+![Assembly5](../../assets/airframes/experimental/stampede/mounting_detail.jpg)
+
+For this particular mounting it was chosen to use the clip supplied with the rover to attach the upper plane. For this, two supports were 3D printed. The CAD files are provided [here](../../assets/airframes/experimental/stampede/plane_holders.zip).
+
+> **Warning** It is **HIGHLY RECOMMENDED** to set the ESC in training mode (see Traxxas Stampede Manual) so to reduce the power to 50%.
+
+
+## Output connections
+
+| PWM Output | Rate | Actuator |
+| -- | -- | -- |
+| MAIN1 | 50 Hz | Not connected |
+| MAIN2 | 50 Hz | Steering servo |
+| MAIN3 | 50 Hz | Not Connected |
+| MAIN4 | 50 Hz | ESC input |
+| MAIN5 | 50 Hz | Not Connected |
+| MAIN6 | 50 Hz | Not Connected |
+| MAIN7 | 50 Hz | Not Connected |
+| MAIN8 | 50 Hz | Not Connected |
+
+## Simulation
+
+To test the algorithms, a simulation was used. This can be ran by running 
+
+ ```sh
+ make posix gazebo-rover
+ ``` 
+
+ from the Firmware folder. 
+
+## Usage
+At the current state, the only supported modes are `MISSION`  and `MANUAL` when a RC remote is connected. To use the mission mode, first upload a new mission to the vehicle with QGC. Then, BEFORE ARMING, select `MISSION` and then arm. 
```

Comment:
Do you mean "At the current time, PX4 only supports `MISSION`  and `MANUAL` modes when a RC remote is connected."?